### PR TITLE
refactored and added new k6 test #GCPActive

### PR DESCRIPTION
**Pull request by acn-sbuad**
## Related Issue(s)
- #267

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)

### CODE REVIEW

File: test/k6/src/api/notifications/orders.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import http from "k6/http";
+
+import * as config from "../../config.js";
+
+import * as apiHelpers from "../../apiHelpers.js";
+
+export function getById(id, token) {
+    var endpoint = config.notifications.orders_fromId(id);
+    return getByUrl(endpoint, queryParams, token);
```
Comment (SandGrainOne) [inline test/k6/src/api/notifications/orders.js:9]: queryParams is not defined here, is it a global? Do we want globals?

File: test/k6/src/api/notifications/orders.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    var params = apiHelpers.buildHeaderWithBearer(token);
+    var response = http.get(url, params);
+
+    return response;
+  }
+
+  export function getBySendersReference(sendersReference, token) {
+    var endpoint =
+      config.notifications.orders_fromSendersRef(sendersReference);
+    var params = apiHelpers.buildHeaderWithBearer(token);
```
Comment (SandGrainOne) [inline test/k6/src/api/notifications/orders.js:22]: Headers called `params`?

File: test/k6/src/api/notifications/orders.js (hunk 1 above)
Comment (acn-sbuad) [reply test/k6/src/api/notifications/orders.js:9]: Nope, gammel moro fra annet api. fikser.

File: test/k6/src/api/notifications/orders.js (hunk 2 above)
Comment (acn-sbuad) [reply test/k6/src/api/notifications/orders.js:22]: thats the practice we've followed for all apis, but we could rename it to headers. params is the name used for the property in the http function we are calling.

**Review by SandGrainOne — APPROVED** [review]

**Pull request merged**